src/role_upgrader.js

```diff
@@ -14,7 +14,7 @@ roles.upgrader.stayInRoom = true;
 roles.upgrader.buildRoad = false;
 roles.upgrader.killPrevious = true;
 
-roles.upgrader.boostActions = ['upgradeController'];
+//roles.upgrader.boostActions = ['upgradeController'];
```

Comment:
How else to enable boosting for healcreeps without boost code messing with cpu for all upgraders?